Makefile

```diff
@@ -94,12 +94,14 @@ depend: init
 
 depend.ensure: init
 
+vendor/Gopkg.lock: Gopkg.lock
+	cp Gopkg.lock vendor/Gopkg.lock
+
```

Comment:
there is nothing depending on this target

Reply:
abandoning this review - it is the wrong approach.  Thanks for the review @rshriram .